packages/caravan-psbt/src/psbtv2/psbtv2.ts

```diff
@@ -833,6 +897,8 @@ export class PsbtV2 extends PsbtV2Maps {
     redeemScript,
     witnessScript,
     bip32Derivation,
+    rbf = false, // Add an optional RBF (Replace-By-Fee) parameter with a default value of false to signal fees replacement
```

Comment:
I'd prefer if this method arg were eliminated. I can't see a good way of handling it without bringing in more complexity surrounding bip125. As the changes are written, this value overrides the sequence which it should not do. Also, the args here are all documented in bip370 alone.

I think it'd be best to just leave it up to the caller to provide a sequence number that will signal for rbf. Perhaps just a comment on the method telling the user where to look.

However, what might have been overlooked in this method is that the `PSBT_IN_SEQUENCE` should be initialized to `0xffffffff` when it's empty and only the locktime is being changed.

Reply:
removed ...